packages/@apphosting/adapter-angular/src/utils.ts

```diff
@@ -143,10 +143,19 @@ function extractManifestOutput(output: string): string {
   return stripAnsi(output.substring(start, end + 1));
 }
 
-export function populateMetadata(angularVersion?: string): Metadata {
-  const packageJson = JSON.parse(readFileSync(`${__dirname}/../package.json`, "utf-8"));
+/**
+ * Create metadata needed for outputting adapter and framework metrics in bundle.yaml.
+ * Includes adapter name, adapter version, framework name and framework version.
+ */
+export function createMetadata(angularVersion?: string): Metadata {
+  const packageJsonPath = `${__dirname}/../package.json`;
+  if (!existsSync(packageJsonPath)) {
+    throw new Error(`Angular adapter package.json file does not exists at ${packageJsonPath}`);
+  }
+  const packageJson = JSON.parse(readFileSync(packageJsonPath, "utf-8"));
   return {
-    adapterNpmPackageName: packageJson.name.replace(/([@])/g, "\\$1"), // escape @ for yamlStringify
+    // escape @ for yamlStringify, e.g. "@apphosting/adapter-angular" -> "\@apphosting/adapter-angular"
```

Comment:
yaml.Stringify will add double quotes on adapter name because of the @:
```
metadata:
  adapterNpmPackageName: "@apphosting/adapter-angular"
  adapterVersion: 17.2.7
```
Now I rethink about this, even tho it looks slightly different from other fields, the publisher will handle this string still fine. Removing the escape here

Reply:
imo I think we should try to keep the fields consistent and still escape the @. Especially once we open this up to more community adapters it may make analysis more complicated if we have some fields with extra quotes and some without.

Reply:
How does the yaml deserialization work on the go side? does the deserialized value include the quotes?

Reply:
I just double checked it in go and the deserialized value doesn't actually include the quotes. Good catch Tony.

Sorry for the back and forth, but that means that we should be good to not escape the @ then and have the quotes as is